styles.css

```diff
@@ -304,7 +311,7 @@ button:disabled {
   display: inline-block;
 }
 
```

Comment:
Please use the font for the paragraph of the `About me` section from the design page.